features/fout-meldingen.feature

```diff
@@ -0,0 +1,220 @@
+        | status | 400                                               |
+        | detail | Parameter 'locatie' is niet van het correcte type |
+        En bevat de response de volgende invalidParams
+        | name    | reason                                                                   |
+        | locatie | waarde '194189.98399146,465870.76799701' is niet in 'epsg:28992' formaat |
+
+Rule: waarde van parameter is valide
+
+    Scenario: waarde van één parameter is niet valide
+        Als '/adresseerbareobjecten/0599040000' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                                      |
+        | title  | Een of meerdere parameters zijn niet correct.               |
+        | status | 400                                                         |
+        | detail | Parameter 'adresseerbaarobjectidentificatie' is niet valide |
+        En bevat de response de volgende invalidParams
+        | name                             | reason                                                   |
+        | adresseerbaarobjectidentificatie | waarde '0599040000' voldoet niet aan patroon ^[0-9]{16}$ |
+
+    Scenario: waarde van meerdere parameters zijn niet valide
+        Als '/adressen?pandIdentificatie=0345100002016017&page=-1&pageSize=2000' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                        |
+        | title  | Een of meerdere parameters zijn niet correct. |
+        | status | 400                                           |
+        | detail | Parameters 'page', pageSize' zijn niet valide |
+        En bevat de response de volgende invalidParams
+        | name     | reason                                 |
+        | page     | waarde '-1' is lager dan minimum 1     |
+        | pageSize | waarde '2000' is hoger dan maximum 100 |
+
+Rule: Een raadpleeg collectie aanroep mag slechts één identificatie parameter bevatten
+
+    Abstract Scenario: Er zijn meerdere identificatie parameters opgegeven
+        Als '<path><query string>' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                        |
+        | title  | Een of meerdere parameters zijn niet correct. |
+        | status | 400                                           |
+        | detail | <fout detail>                                 |
+
+        Voorbeelden:
+        | path                   | query string                                                                                      | fout detail                                                                                                |
+        | /adressen              | ?pandIdentificatie=0226100000008856&adresseerbaarObjectIdentificatie=0226010000038820             | Alleen parameter 'pandIdentificatie' of 'adresseerbaarObjectIdentificatie' moet zijn opgegeven             |
```

Comment:
Moet de fout detail niet meer generiek zijn?
Dus:

Alleen parameter 'zoekresultaatidentificatie', 'pandIdentificatie' of 'adresseerbaarObjectIdentificatie' moet zijn opgegeven